### ISSUE TITLE
Could not install with --HEAD --with-xwidgets

### ISSUE THREAD

**Issue by d12frosted** [d12frosted/homebrew-emacs-plus#199]
**Describe the bug**
`emacs-plus` fails to install

**Command used to install `emacs-plus`**

```
$ brew install emacs-plus --HEAD --with-xwidgets
```

**Installation log**

```
Undefined symbols for architecture x86_64:
  "_CHECK_RANGED_INTEGER", referenced from:
      _Fxwidget_webkit_goto_history in xwidget.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[1]: *** [temacs] Error 1
```

Full log can be found in the [CI][1].

  [1]: https://github.com/d12frosted/homebrew-emacs-plus/runs/566771546#step:3:245

**Comment by mdoza** [comment]
@d12frosted `brew install emacs-plus --HEAD --with-xwidgets --with-emacs-27-branch`

**Comment by d12frosted** [comment]
@mdoza can you please elaborate a little bit more? What do you mean by your comment?

**Comment by mdoza** [comment]
> @mdoza can you please elaborate a little bit more? What do you mean by your comment?

Of course. Presumably you would like to get xwidgets working; install with `—with-emacs-27-branch` along with `—HEAD` and you should be in business.

**Comment by d12frosted** [comment]
Oh cool. Good to know. Thank you!

**Comment by d12frosted** [comment]
Fixed via #201
